### DISCUSSION TITLE
Why I can't change my photo profile

### DISCUSSION THREAD

**Discussion by MiftahAlbi** [community/community#65251] [New to GitHub]
Why I can't change my photo profile?

**Comment by Imen-Frigui** [comment]
First go to setting click on Edit on the photo then click the green button update profile.

Note:  -"Your profile picture should be a PNG, JPG, or GIF file, and it must be less than 1 MB in size and smaller than 3000 by 3000 pixels. For the best quality rendering, we recommend keeping the image at about 500 by 500 pixels."

This could help: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile

**Reply by MiftahAlbi** [reply]
No option for edit profile and also green button

**Reply by queenofcorgis** [reply]
@MiftahAlbi it may be helpful to take some screenshots (redacting any PII or otherwise private info) so we can better assist!

**Comment by Arkenstine** [comment]
Maybe try a different browser? I've had this problem before on Safari, but it worked when i tried it on chrome
